### ISSUE TITLE
4.1.2 Name, Role, Value: clarify that non-web software does not have a user agent

### ISSUE THREAD

**Issue by mitchellevan** [w3c/wcag2ict#605]
### Currently

[4.1.2 in WCAG2ICT](https://www.w3.org/TR/wcag2ict/#name-role-value) does not provide word substitution for this phrase:

> notification of changes to these items is available to user agents, including assistive technologies

This is correct for non-web documents, but non-web software cannot have a user agent.

### Proposal 1

Word substitution:

> notification of changes to these items is available to **user agents or platform software**, including assistive technologies

And add a new note, explaining that "user agents" is for non-web documents and "platform software" is for non-web software.

### Proposal 2

On second thought, this is probably better:

> notification of changes to these items is available to **assistive technologies and accessibility features of software**

This matches the word substitution in "[programmatically determinable](https://www.w3.org/TR/wcag2ict/#dfn-programmatically-determinable)" and a couple of other places.

**Comment by ChrisLoiselle** [comment]
Notes from June 5, 2025 meeting:

- We also use "user agent" or "platform software" and our definition of platform software includes user agent. May need to tweak the definition
- User agent / platform software. They should be defined as different terms. Platform software can act as a user agent.
- User agent is usually an agent for some type of content (e.g. for web, or for some type of document).

**Comment by maryjom** [comment]
@ChrisLoiselle I don't want to change the definition. User agents can be a platform (e.g. a web browser), as defined. We also have a definition of user agent - which pertains to documents. It's just that we defined a user agent to be an example of platform software, so where we use "user agent or platform software" it should be something more like "platform software, including user agents" or "platform software, such as a user agent". I think the former example is already being used in some places and needs to be used more consistently.

**Comment by GreggVan** [comment]
Ouch

Hate to reopen this but. 

I'm pretty sure this should have been     "assistive technologies and accessibility features of **platform software below**”


it is only  (AT and ) the accessibility features of the platform software below the software  that they need to work with and that they can rely on.   accessibility features of other software is not relevant.    and their own accessibility features …. well presumably they are already compatible with those.

**Comment by maryjom** [comment]
@GreggVan IMO, I don't think that one needs to get overly prescriptive and wordy on this. Platform software would be, naturally, the layer below the software trying to meet this criterion.

**Comment by maryjom** [comment]
Closing. The changes from the [Resolution on 10 July](https://www.w3.org/2025/07/10-wcag2ict-minutes.html#a4c3) with the one editorial correction post-meeting are merged into the editor's draft.
